main.py

```diff
@@ -3245,7 +3245,11 @@ def update_weather_data(
             if temp_data and temp_data.lower() != 'none':
                 temperature = db.weather_processor._convert_temperature_unit(temp_data)
             else:
-                temperature = '--°'
+                target_unit = config_center.read_conf('Weather', 'temperature_unit', 'celsius')
+                temperature = '--℃'
+                if target_unit == 'fahrenheit':
+                    temperature = '--℉'

```

Comment:
这很优雅吗
```suggestion
                temperature = '--°F' if target_unit == 'fahrenheit' else '--°C'
```

而且貌似整个 cw 里面 `℃` `℉` 和 `°C` `°F` 是混用的 @baiyao105

Reply:
> 这很优雅吗

改后确实更优雅
~~我们应该不会考虑开尔文作为温标的情况吧~~

Reply:
不考虑

写法的统一问题不是这个 pr 应该干的

你先随便写一种

如果你想水一个 pr 可以另开一个统一一下

Reply:
好的